package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "Stethoscope",
-  "version": "3.0.6",
+  "version": "3.1.0",
   "private": true,
-  "homepage": "https://stethoscope.app",
```

Comment:
oops, did we want to keep this `./` instead of my change to the URL?

Reply:
It needs to be `./` because in the URL in that field is used to replace `%PUBLIC_URL%` in the index.html template file. Forgot that until I built the app and saw that nothing was loading.